doc/spectrographs/gemini_gmos.rst

```diff
@@ -118,3 +123,35 @@ your RMS is a factor of 2-3 too high, consider setting:
             fwhm_fromlines = False
 
 
+MultiSlit
+=========
+
+Mask Definition
+---------------
+
+PypeIt can now take advantage of the mask definition file
+generated when one designs a GMOS mask.  To do so, one needs
+to provide two additional files and specify them 
+with the :doc:`pypeit_file`:
+
+#.  The mask definition file, aka ODF file
```

Comment:
"This is the output generated from GMMPRS. Must be in the FITS format. We do not support ASCII mask files currently."

Reply:
done